Makefile

```diff
@@ -2,10 +2,8 @@ BUILD_TAG ?= latest
 BUILD_TAG_ALPINE ?= latest-alpine
 BUILD_CACHE_TAG = latest-builder-cache
 IMAGE_NAME = kubernetes-fluentd
-ECR_URL = public.ecr.aws/u5z5f8z6
+ECR_URL = public.ecr.aws/sumologic
```

Comment:
Can we stick to `OPENSOURCE_ECR_URL` name as we did in `sumologic-otel-collector`?

https://github.com/SumoLogic/sumologic-otel-collector/blob/main/Makefile#L83

Reply:
I don't see why we would.

Reply:
What do you mean? I believe it would be nice to stick to the same "naming scheme" throughout the repos that we maintain

Reply:
I agree we should stick to a naming scheme. Can you explain why you think `OPENSOURCE_ECR_URL` is better than `ECR_URL` or even `REGISTRY_URL`?

Reply:
Well I believe we should keep `ECR` in the name as the actions around it are in some way specific to AWS ECRs.

In terms of adding `OPENSOURCE_` prefix I believe it gives the the clear intention what's the purpose of this ECR or what sort of artifacts does it contain. At least IMHO.

Reply:
FYI: In tools it's currently just `ECR_URL`: https://github.com/SumoLogic/sumologic-kubernetes-tools/blob/5b44191084c888df4986664df0ada81d5d3759c0/Makefile#L5

Reply:
IMO, all these names are fine and clearly express their intent given that we only want to use one ECR registry, the important part is that we're consistent across projects. Given that tools has `ECR_URL`, I'm fine with using that here, and everywhere else.

Reply:
@pmalek-sumo that ok with you?

Reply:
This has been already merged and this PR is not needed anymore I believe

Reply:
https://github.com/SumoLogic/sumologic-kubernetes-fluentd/pull/379